docs/nitclk/class.rst

```diff
@@ -183,7 +164,7 @@ finish_sync_pulse_sender_synchronize
             
 
 
-        :type sessions: list of (nimi-python Session class or nitclk.SessionReference)
+        :type sessions: list of (Driver Session or nitclk.SessionReference)
```

Comment:
I think we should keep the old version - `nimi-python Session class`. @marcoskirsch, can you please confirm.

Reply:
This was a case in which the config file had duplicate entries, so I removed one of them (based on consistency with the rest of the functions in the API, but I'm not completely sure). Note the functions.py diff:

![image](https://user-images.githubusercontent.com/80292064/121231365-ef730500-c855-11eb-8eda-5fce10063312.png)

Reply:
>I think we should keep the old version - nimi-python Session class. @marcoskirsch, can you please confirm.

The average person calling `pip install niscope` has no good reason to be familiar with the term `nimi-python`. 99% of the clients using our Python APIs are going to get them from PyPI. I actually want to open an Issue about the fact that https://pypi.org/project/niscope/ talks about nimi-python and mentions things beyond `niscope`.

All this said, type_in_documentation can be `nifgen.Session`, `niscope.Session`, `nidigital.Session`, `niscope.Session` _or_ `nitclk.SessionReference`. Not sure what the best way to refer to these generically is.

Reply:
That makes sense.

We have used "Session instance" [elsewhere](https://nimi-python.readthedocs.io/en/master/search.html?q=%22Session+instance%22&check_keywords=yes&area=default#) in documentation. But here we also need to differentiate between nitclk and other modules. How about:
```
'list of instrument Session or nitclk SessionReference instances'
```

Reply:
yes

Reply:
minor tweak

'list of instrument-specific Session or nitclk.SessionReference instances'

Reply:
Ok, pushed new changes based on [updated](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/172689) metadata.